package.json

```diff
@@ -86,17 +86,19 @@
   },
   "dependencies": {
     "core-js": "^2.5.3",
+    "custom-event-polyfill": "^1.0.6",
     "draft-js": "0.10.5",
     "draftail": "^1.2.1",
     "element-closest": "^2.0.2",
     "focus-trap-react": "^3.1.0",
-    "prop-types": "^15.6.0",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.4.0",
```

Comment:
16.8.6 is the latest version

Reply:
Indeed, but it’s extremely tricky to make React updates on Wagtail. It was already a pain to just go from React 16.2 to 16.4 and make sure everything else still built. At first I tried to build with 16.7, but there was a chain reaction of dependencies updates that implied a lot of rewrite of other modules, so I had to keep an “old” version for now.

Reply:
OK fair enough.